src/plugins/azgenerator/azgenerator.ts

```diff
@@ -10,8 +10,17 @@ export async function processRequest(host: Host) {
     //host.Message({Channel:Channel.Warning, Text:"in azgenerator processRequest"});
     try {
         const session = await startSession<CodeModel>(host, {}, codeModelSchema);
-        let cliSettings = await session.getValue('az');
-        let testScenario: any[] = cliSettings['test-setup'] || cliSettings["test-scenario"];
+        let testScenario: any[] = [];
+        try {
+            let cliSettings = await session.getValue('az');
+            testScenario = cliSettings['test-setup'] || cliSettings["test-scenario"];
+            if (!testScenario) {
+                cliSettings = await session.getValue('cli');
+                testScenario = cliSettings['test-setup'] || cliSettings["test-scenario"];
+            }
```

Comment:
according to our discussion. the test-setup or test-scenario should be outside of az or cli scope. In that way, This should be `testScenario = await session.getValue('test-setup') ||  await session.getValue('test-scenario');`

Reply:
Thanks Qiaoqiao. 
It seems will be incompatible with some readme.cli.md where  test-scenario has been configured under 'cli'. 
So it means we will change all those current configuration in readme.cli.md?

Reply:
I think those settings only affect poc and az now. you can change poc code and az code simultaneously. But you can confirm with Zim what else will be affected.

Reply:
Hi @zikalino , Just want to confirm where shall we put the 'test-scenario' configuration in readme.cli.md？

Will it be under root configuration like
A)
```
``` yaml $(cli)
cli:
  namespace: azure.mgmt.xxx
test-scenario:
  - name: Factories_CreateOrUpdate
```
or embeded in the 'cli' object ?
B)
```
``` yaml $(cli)
cli:
  namespace: azure.mgmt.xxx
  test-scenario:
    - name: Factories_CreateOrUpdate
```

As I check the current configurations, it seems we are using solution B， will we change it to solution A?